README.md

```diff
@@ -1 +1,61 @@
-# homework-microservice
\ No newline at end of file
+# Homework Microservice
+
+## Description
+
+The Homework domain manages all information related to assignments for each course, including workflows for completing homework, submission tracking, and grading. Staff can post workflows to guide students through the homework process, which enhances the user experience (UX) for both students and staff.
+
+## Setup and Usage
+
+### Prerequisites
+
+- **Go** (>= 1.19)
+- **tmux** (optional, required only for running the bash script)
+
+### Running the Microservice
+
+#### 1. Run the Service Using the Makefile
+
+- Start the server and client:
+  ```bash
+  make

```

Comment:
This currently doesnt run the files, no? The makefile has one rule over compiling protos.